views/verify_email.ejs

```diff
@@ -82,7 +82,7 @@
         <p class="email-message">
           You are welcome to Aphia! We are glad to have you onboard. Kindly
           click on this link to verify your account
-          <a href="https://aphia.onrender.com/api/verify/?email=<%=email %>">Click here</a>
+          <a href="https://blackholemain.onrender.com/api/verify/?email=<%=email %>">Click here</a>
```

Comment:
Same here. point to the front end url

Reply:
we need to test it from the backend first